src/arch/riscv/kernel/context.h

```diff
@@ -1,7 +1,35 @@
 #ifndef ARCH_RISCV_CONTEXT_H_
 #define ARCH_RISCV_CONTEXT_H_
 
+#ifndef __ASSEMBLER__
+
 struct context {
+	unsigned int ra;
+	unsigned int sp;
+	unsigned int s[12];
+	unsigned int mstatus;
 };
 
+#else
```

Comment:
#endif ?
Do you want to hide definitions in none assembly code?

Reply:
Not really. Fixup done.